data/json/items/comestibles/dairy.json

```diff
@@ -6,7 +6,7 @@
     "copy-from": "milk",
     "spoils_in": "12 hours",
     "price": 19,
-    "contamination": 5,
+    "contamination": [ [ "bad_food", 5 ] ],
```

Comment:
Could I suggest you use an array of objects instead of an array of arrays?
Something like
`[ { "bad_food": 5 } ]`
or
`[ { "disease": "bad_food", "amount": 5 } ]`